lib/reporters/base.js

```diff
@@ -56,6 +56,11 @@ exports.useColors =
 
 exports.inlineDiffs = false;
 
+/**
+ * Truncate diffs longer than this value to avoid slow performance
+ */
+exports.maxDiffSize = 2048;
```

Comment:
Setting it to a NaN would lead to no limit on the diff size, does not seem right...
I'll change this so that NaN has no effect, and the default value will be used instead.

Reply:
Ok. Make sure that no error can be thrown in `base.js`, whatever user's input could be.
Mocha's reaction to errors thrown in a reporter is suboptimal.

Reply:
I tested and `NaN` still results in no limit.

Reply:
My bad, fixed now